Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -896,6 +913,17 @@ private void GetComponentByTypeAndName(Type type, string managerName, out IMixed
 
             manager = null;
 
+            if (isMixedRealityManagerInitializing)
+            {
+                Debug.LogWarning("Unable to get manager a manager while initializing!");
+                return;
+            }
+
+            if (mixedRealityComponentsCount != MixedRealityComponents.Count)
```

Comment:
Why do we keep a count reference that's expected to stay equal to `MixedRealityComponents.Count`? Seems like duplicate data.